apps/web/playwright/fixtures/users.ts

```diff
@@ -376,6 +376,15 @@ export const createUsersFixture = (page: Page, emails: API | undefined, workerIn
       await prisma.user.delete({ where: { id } });
       store.users = store.users.filter((b) => b.id !== id);
     },
+    set: async (email: string) => {
```

Comment:
Better name could be `addExistingUserWithEmail`. There is a `get` method that returns all users, so `set` should be reserved for a complimentary job to that.